### PR TITLE
String manipulation 1

### PR DESCRIPTION
**Pull request by Matthew-Abebe**
Screenshot (212)
Screenshot (213)

### CODE REVIEW

File: string-manipulation-1/src/get-words.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+function getWords(string) {
+
+  var arr = [];
+
+  if (string === '') {
+    return arr;
+  }
+
+  for (var i = 0; i < string.length; i++) {
+    var word = string.split(' ');
```
Comment (codingcodymiller) [inline string-manipulation-1/src/get-words.js:12]: The `split` method already uses a loop for you, this loop is unnecessary. I recommend you tinker with `split` and see what steps are actually necessary here.

File: string-manipulation-1/src/is-vowel.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* exported isVowel */
+
+function isVowel(char) {
+  if (char === 'A' || char === 'E' || char === 'I' || char === 'O' || char === 'U' || char === 'a' || char === 'e' || char === 'i' || char === 'o' || char === 'u') {
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-vowel.js:4]: If you convert the letter to either `upperCase` or `lowerCase`, this condition can be reduced by half.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start here! Please see my notes

File: string-manipulation-1/src/is-vowel.js (hunk 2 above)
Comment (Matthew-Abebe) [reply string-manipulation-1/src/is-vowel.js]: Screenshot (234)
Screenshot (235)

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**